src/machine/i2c.go

```diff
@@ -1,5 +1,5 @@
-//go:build atmega || nrf || sam || (stm32 && !stm32wlx) || fe310 || k210 || rp2040
-// +build atmega nrf sam stm32,!stm32wlx fe310 k210 rp2040
+//go:build atmega || nrf || sam || fe310 || k210 || rp2040 || stm32wlx
```

Comment:
This will disable for all other stm32?  Should just leave the stm32 clause?

Reply:
Was a mistake. fixed